package.json

```diff
@@ -4,21 +4,22 @@
   "description": "It's cool when users own their content.",
   "main": "index.js",
   "scripts": {
-    "build": "./node_modules/.bin/gulp",
+    "build": "gulp",
```

Comment:
I understand the intention but `npm run` puts the `.node_modules/.bin/` path at the front of your PATH when using scripts so it's not necessary to be explicit. https://docs.npmjs.com/misc/scripts#path

You can verify this with this script: `"which": "echo $PATH"`